### PR TITLE
Add pangram exercise

### PR DESCRIPTION
**Pull request by keiravillekode**

### CODE REVIEW

File: exercises/practice/pangram/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Instructions append
+
+## Registers
+
+| Register | Usage     | Type    | Description                                       |
+| -------- | --------- | ------- | ------------------------------------------------- |
+| `$a0`    | input     | address | null-terminated input string                      |
+| `$v0`    | output    | boolean | input is an pangram (`0` = `false`, `1` = `true`) |
```
Comment (ErikSchierboom) [inline exercises/practice/pangram/.docs/instructions.append.md:8]:
```suggestion
| `$v0`    | output    | boolean | input is a pangram (`0` = `false`, `1` = `true`) |
```

File: exercises/practice/pangram/impl.mips [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# | Register | Usage     | Type    | Description                                       |
+# | -------- | --------- | ------- | ------------------------------------------------- |
+# | `$a0`    | input     | address | null-terminated input string                      |
+# | `$v0`    | output    | boolean | input is an pangram (`0` = `false`, `1` = `true`) |
```
Comment (ErikSchierboom) [inline exercises/practice/pangram/impl.mips:4]:
```suggestion
# | `$v0`    | output    | boolean | input is a pangram (`0` = `false`, `1` = `true`) |
```

**Review by ErikSchierboom — COMMENTED** [review]
Two teeny grammar nits.

File: exercises/practice/pangram/impl.mips (hunk 2 above)
Comment (keiravillekode) [reply exercises/practice/pangram/impl.mips]: done

File: exercises/practice/pangram/.docs/instructions.append.md (hunk 1 above)
Comment (keiravillekode) [reply exercises/practice/pangram/.docs/instructions.append.md]: done

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
@keiravillekode As you're now a member of the `MIPS` team, feel free to merge this yourselves!

**Pull request merged**